autoload/lsp/hover.vim

```diff
@@ -72,12 +72,14 @@ enddef
 
 # process the 'textDocument/hover' reply from the LSP server
 # Result: Hover | null
-export def HoverReply(lspserver: dict<any>, hoverResult: any): void
+export def HoverReply(noEcho: bool, lspserver: dict<any>, hoverResult: any): void
```

Comment:
Can you change "noEcho" to "silent" and move this to the last parameter?  Most of the functions take lspserver as the first argument.

Reply:
done.
modified 'noecho' to 'silent'.
// as for arg place order, let's refactor it later.